ibm_catalog.json

```diff
@@ -0,0 +1,66 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-secrets-manager-iam-engine",
+      "label": "IAM secrets engine module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "iam",
+        "secrets",
+        "encryption"
+      ],
+      "short_description": "This module configures an IAM secrets engine for an existing secrets manager instance.",
```

Comment:
The SM docs are not consistent about whether to call this an IAM credentials engine, and IAM credentials secrets engine, or an IAM secrets engine. Seems like IAM credentials engine would be the most clear. Thoughts? (https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine&interface=ui).

Whatever we use in this short_desc should be duplicated in the `settings.yml` file description.

Is is an [IAM credentials engine](url) or a [Secrets Manager secrets engine](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-secrets-engines)?

Reply:
Let's go with `IAM credentials engine` as that seems to be the official name.